### DISCUSSION TITLE
node version not right. How to solve that ?

### DISCUSSION THREAD

**Discussion by RoelofWobben** [leokhoa/laragon#826] [Q&A]
Hello, 

I want to use laragon to practice with making plugins for wordpress.
but when I want to make a new plugin I see this : 

``` 
Minimum system requirements not met!

Error: Wanted node version >=20.10.0 (>=20.10.0)
Error: Wanted npm version >=10.2.3 (>=10.2.3)
``` 

how can I solve this with Win 10 ?

**Comment by Pen-y-Fan** [comment]
Download the pre built binary from [nodejs](https://nodejs.org/en/download/prebuilt-binaries)

e.g. https://nodejs.org/dist/v20.15.1/node-v20.15.1-win-x64.zip

Unzip the downloaded zip file and copy the directory to **C:\laragon\bin\nodejs** folder. You should already have a node-v18 directory.

Rename the new directory **node-v20** (remove the 15.1-win-x64)

In Laragon navigate the Laragon Menu > Node > Version [Node-v18] > choose **Node-v20** (the directory you just created)

Laragon will switch Node versions.

To confirm, open the Laragon Terminal and run these commands:

```sh
node -v

npm -v
```

You should see:

![image](https://github.com/leokhoa/laragon/assets/40126936/a6c423cd-2b8b-46d8-a479-dd1c6167845b)

If you want Node in other terminals, you may need Laragon > Tools > Path > Add Laragon to Path
